src/wiki/getting-started/installing-optifine.md

```diff
@@ -8,7 +8,7 @@ eleventyNavigation:
 
 Because OptiFine is an old, closed-source project, it has evolved to cause many mod incompatibilities, and doesn't do as good of a job as some the alternatives out there.
 
-Also, if you're using the Fabric mod loader, there's no native support for it, and the current [workaround](#fabric) is known to be buggy and quite unstable, due to the very nature of OptiFine and OptiFabric.
+Also, if you're using the Fabric mod loader, there's no native support for it, and the current workaround is known to be buggy and quite unstable, due to the very nature of OptiFine and OptiFabric.
```

Comment:
there are a couple links in this area, which got removed... is that intentional?

Reply:
Those links didn't resolve when I tested it locally, and they were erroring in markdownlint. I'm not sure what else I was supposed to do with them.

Reply:
they are supposed to go to a header on the same page, but idk... maybe they were always fucked

Reply:
they were working on the polymc.org deployment, so it should work here too... 🤔